gcs/conn.go

```diff
@@ -184,21 +184,23 @@ func (c *conn) OpenBucket(
 	// bucket.
 	_, err = b.ListObjects(ctx, &ListObjectsRequest{MaxResults: 1})
 
-	if typed, ok := err.(*googleapi.Error); ok {
-		switch typed.Code {
-		case http.StatusForbidden:
-			err = fmt.Errorf(
-				"Bad credentials for bucket %q. Check the bucket name and your "+
-					"credentials.",
-				b.Name())
-
-			return
-
-		case http.StatusNotFound:
-			err = fmt.Errorf("Unknown bucket %q", b.Name())
-			return
-		}
-	}
+	var apiError *googleapi.Error
+	if errors.As(err, &apiError) {
+		switch apiError.Code{
```

Comment:
Please configure your editor or IDE to use `gofmt` when saving files. 

This file is not correctly formatted right now, please use gofmt to fix it.

Reply:
fixed